web/src/main/java/it/pagopa/selfcare/dashboard/web/model/UserResource.java

```diff
@@ -0,0 +1,33 @@
+package it.pagopa.selfcare.dashboard.web.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
```

Comment:
@KevinSi96 use @Data instead of @Getter and @Setter so Lombok will create the toString() method too